libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -7,21 +7,22 @@
       placeholder="Search for books to add to your reading list"
       formControlName="term"
     />
-    <button mat-icon-button matSuffix>
+    <button mat-icon-button matSuffix
+      [attr.aria-label]="'Search'">
```

Comment:
Is the "attribute binding" syntax really needed here, for plain text?

Reply:
I have removed attribute aria-label for the buttons having plain-text, but aria-label is needed here as this button doesn’t have plain-text. I checked removing it but it shows accessibility issue after running light-house extension.